hafnian/tests/test_torontonian.py

```diff
@@ -64,4 +106,11 @@ def test_torontonian_vacuum():
     n_modes = 5
     Omat = np.zeros([2 * n_modes, 2 * n_modes])
     tor_val = tor(Omat)
-    assert np.abs(tor_val.real - 0.0) < abs_tol
+    assert np.allclose(tor_val.real, 0.0)
+
+
+@pytest.mark.parametrize("l", [1, 2, 3, 4, 5])
+@pytest.mark.parametrize("nbar", np.arange(0.25, 3, 0.25))
+def test_torontononian_analytical_mats(l, nbar):
+    """Checks the correct value of the torontonian for the analytical family described by gen_omats"""
+    np.allclose(torontonian_analytical(l, nbar), tor(gen_omats(l, nbar)))
```

Comment:
```suggestion
    assert np.allclose(torontonian_analytical(l, nbar), tor(gen_omats(l, nbar)))
```

Reply:
Should this test have `atol` and `rtol`?

Reply:
The defaults are fine